legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -498,22 +498,22 @@ protected static Date toDateIfNotNull(Instant instant)
         return (instant == null) ? null : Date.from(instant);
     }
 
-    protected static Integer parseIntegerIdIfNotNull(String id)
+    protected static Long parseNumericIdIfNotNull(String id)
     {
-        return parseIntegerIdIfNotNull(id, null);
+        return parseNumericIdIfNotNull(id, null);
     }
 
-    protected static Integer parseIntegerIdIfNotNull(String id, Status errorStatus)
+    protected static Long parseNumericIdIfNotNull(String id, Status errorStatus)
     {
-        return (id == null) ? null : parseIntegerId(id, errorStatus);
+        return (id == null) ? null : parseNumericId(id, errorStatus);
     }
 
-    protected static int parseIntegerId(String id)
+    protected static long parseNumericId(String id)
     {
-        return parseIntegerId(id, null);
+        return parseNumericId(id, null);
     }
 
-    protected static int parseIntegerId(String id, Status errorStatus)
+    protected static long parseNumericId(String id, Status errorStatus)
```

Comment:
Don't you need to actually change the function to use Long.parseLong instead of Integer.parseInt?